journey-map/public/index.html

```diff
@@ -91,23 +96,52 @@ <h1>IPV Core Journey Map</h1>
 
             // Set up the form options
             const { disabledOptions, featureFlagOptions } = getOptions(journeyMap);
-            disabledOptions.forEach((option) => {
-                const element = document.createElement('option');
-                element.value = option;
-                element.innerText = option;
-                disabledInput.appendChild(element);
-            });
-            featureFlagOptions.forEach((option) => {
-                const element = document.createElement('option');
-                element.value = option;
-                element.innerText = option;
-                featureFlagInput.appendChild(element);
-            });
+            if (disabledOptions.length > 0) {
+                disabledOptions.forEach((option) => {
+                    const input = document.createElement('input');
+                    input.type = 'checkbox';
+                    input.name = 'disabledCri';
+                    input.value = option;
+                    input.id = option;
+
+                    const label = document.createElement('label');
+                    label.htmlFor = option;
+                    label.innerText = option;
+
+                    const span = document.createElement('span');
+                    span.appendChild(label);
+                    span.appendChild(input);
+                    disabledInput.appendChild(span);
+                });
+            } else {
+                disabledInput.append("N/A")
+            }
+
+            if (featureFlagOptions.length > 0) {
+                featureFlagOptions.forEach((option) => {
+                    const input = document.createElement('input');
+                    input.type = 'checkbox';
+                    input.name = 'flag';
+                    input.value = option;
+                    input.id = option;
+
+                    const label = document.createElement('label');
+                    label.htmlFor = option;
+                    label.innerText = option;
+
+                    const span = document.createElement('span');
+                    span.appendChild(label);
+                    span.appendChild(input);
+                    featureFlagInput.appendChild(span)
+                });
+            } else {
+                featureFlagInput.append("N/A");
+            }
+
 
             // Render the journey map SVG
             const renderSvg = async (formData) => {
-                const options = Object.fromEntries(formData || []);
```

Comment:
Stopped converting to an object here, as when you have multiple checkboxes selected (with the same `name` attribute), you lose all values except for the last one. The key in the object can only hold one value. The `FormData` object maintains an array of values so we know what was selected.